StyleCop.Analyzers/StyleCop.Analyzers.Test.CSharp9/NamingRules/SA1312CSharp9UnitTests.cs

```diff
@@ -3,9 +3,34 @@
 
 namespace StyleCop.Analyzers.Test.CSharp9.NamingRules
 {
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.CodeAnalysis.Testing;
     using StyleCop.Analyzers.Test.CSharp8.NamingRules;
+    using Xunit;
+    using static StyleCop.Analyzers.Test.Verifiers.StyleCopCodeFixVerifier<
+        StyleCop.Analyzers.NamingRules.SA1312VariableNamesMustBeginWithLowerCaseLetter,
+        StyleCop.Analyzers.NamingRules.RenameToLowerCaseCodeFixProvider>;
 
     public partial class SA1312CSharp9UnitTests : SA1312CSharp8UnitTests
     {
+        [Fact]
+        [WorkItem(3977, "https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3977")]
+        public async Task TestLambdaDiscardParametersDoNotReportAsync()
+        {
+            var testCode = @"
+using System;
+
+public class TestClass
+{
+    public void Test()
+    {
+        Func<int, int, int> handler = (_, _) => 0;
```

Comment:
Is this a relevant test? Won't this analyze "handler", since that is the only variable?